data/sql_updates/large_aggregates.sql

```diff
@@ -128,49 +85,47 @@ pac_totals as (
-        extract(year from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as year,
-        sum(coalesce(ttl_receipts, 0) -
+        extract(month from coverage_end_date) as month,
+        extract(year from coverage_end_date) as year,
+        sum(coalesce(receipts, 0) -
             (
-                coalesce(pol_pty_cmte_contb_per_i,0) +
-                coalesce(other_pol_cmte_contb_per_i,0) +
-                coalesce(offests_to_op_exp,0) +
-                coalesce(fed_cand_contb_ref_per,0) +
-                coalesce(tranf_from_nonfed_acct_per,0) +
+                coalesce(political_party_committee_contributions,0) +
+                coalesce(other_political_committee_contributions,0) +
+                coalesce(offsets_to_operating_expenditures,0) +
+                coalesce(fed_candidate_contribution_refunds,0) +
+                coalesce(transfers_from_nonfed_account,0) +
+                -- this was already commented out, want to confirm this
                 -- coalesce(loan_repymts_received_per,0) +
-                coalesce(loan_repymts_made_per, 0) +
-                coalesce(ttl_contb_refund,0)
+                coalesce(loan_repayments_other_loans, 0) +
+                coalesce(contribution_refunds,0)
             )
         ) as party_adjusted_total_receipts,
-        sum(coalesce(ttl_disb,0) -
+        sum(coalesce(disbursements,0) -
             (
-                coalesce(shared_nonfed_op_exp_per,0) +
+                coalesce(shared_nonfed_operating_expenditures,0) +
                 -- confirm var
-                coalesce(tranf_to_affliliated_cmte_per,0) +
+                coalesce(transfers_to_other_authorized_committee,0) +
                 -- coalesce(tranf_to_other_auth_cmte,0) +
-                coalesce(fed_cand_cmte_contb_per,0) +
-                coalesce(loan_repymts_made_per,0) +
-                coalesce(ttl_contb_refund,0) +
-                coalesce(other_disb_per,0)
+                coalesce(fed_candidate_committee_contributions,0) +
+                coalesce(loan_repayments_other_loans,0) +
+                coalesce(contribution_refunds,0) +
+                coalesce(other_disbursements,0)
             )
         ) as party_adjusted_total_disbursements
-    from fec_vsum_f3x_vw
-    left join
-        ofec_committee_detail_mv_tmp on committee_id = cmte_id
+    from ofec_totals_parties_mv_tmp
     where
-        most_recent_filing_flag like 'Y'
-        and ofec_committee_detail_mv_tmp.committee_type in ('X', 'Y')
-        and ofec_committee_detail_mv_tmp.designation <> 'J'
+        committee_type in ('X', 'Y')
+        and designation <> 'J'
         -- excluding host conventions because they have different rules than party committees
-        and cmte_id not in ('C00578419', 'C00485110', 'C00422048', 'C00567057', 'C00483586', 'C00431791', 'C00571133',
+        and committee_id not in ('C00578419', 'C00485110', 'C00422048', 'C00567057', 'C00483586', 'C00431791', 'C00571133',
```

Comment:
MIght make sense in the future to factor these into a table for maintainability.  But this works.